docs/installation/server-config/integrate-ksql-with-confluent-control-center.rst

```diff
@@ -93,8 +93,24 @@ that the browser can resolve through externally available DNS.
 
 ::
 
-    confluent.controlcenter.<ksql-cluster-name>.ksql.url=<private-url1>, <private-url2>, ...
-    confluent.controlcenter.ksql.<ksql-cluster-name>.advertised.url=<public-url1>, <public-url2>, ...
+    confluent.controlcenter.ksql.<ksql-cluster-name>.url=<internally-resolvable-hostname1>, <internally-resolvable-hostname2>, ...
+    confluent.controlcenter.ksql.<ksql-cluster-name>.advertised.url=<externally-resolvable-hostname1>, <externally-resolvable-hostname2>, ...
+
+Use the ``curl`` command to check whether these URLs are reachable. Depending
+on your deployment, you may need to check from two different hosts: 
+
+* Check from the host where the browser is running, which is relevant for the
+  ``confluent.controlcenter.ksql.<name>.advertised.url`` setting.
+* Check from the host where |c3-short| is running, which is relevant 
+  for the ``confluent.controlcenter.ksql.<name>.url`` setting.
+
+On both hosts, run the following command to confirm that the KSQL Server
+cluster is reachable.
+
+.. codewithvars:: bash
+
+   curl http://<ksql-hostname>:8088/info \
+   {"KsqlServerInfo":{"version":"|release|","kafkaClusterId":"<ksql-cluster-name>","ksqlServiceId":"default_"}}%   
```

Comment:
Sorry if I missed it, but I think it's also worth mentioning setting/checking the ksql listeners